src/BuildScriptGenerator/DotNetCore/DotnetCoreBashBuildSnippetProperties.cs

```diff
@@ -37,6 +37,8 @@ public class DotNetCoreBashBuildSnippetProperties
 
         public string ManifestFileName { get; set; }
 
+        public string ManifestFileDir { get; set; }
```

Comment:
maybe this class could inherit from `BaseBashBuildScriptProperties`, or from a new common base class, to avoid this declaration's repetition?

Reply:
The script templates themselves do not inherit each other, so I am not too keen on adding an inheritance hierarchy for their supporting classes. Let me know what you think. Inheritance creates a  rigid model IMO (which was the reason originally for me to move away dotnet core separately as it did not fit nicely with the interpreted languages which the base build script is mostly suited for)

Reply:
fair enough 👍